src/components/dev-hub/chart.js

```diff
@@ -0,0 +1,62 @@
+import React, { useMemo } from 'react';
+import PropTypes from 'prop-types';
+import { buildQueryString } from '../../utils/query-string';
+import styled from '@emotion/styled';
+
+const DEFAULT_CHART_HEIGHT = '570';
+const DEFAULT_CHART_WIDTH = '760';
+const DEFAULT_CHART_THEME = 'dark';
+
+const buildChartUrl = (baseUrl, options) => {
+    const params = {
+        autorefresh: options.autorefresh,
+        id: options.id,
+        theme: options.theme || DEFAULT_CHART_THEME,
+    };
+    const queryString = buildQueryString(params);
+    return `${baseUrl}/embed/charts${queryString}`;
+};
+
+const StyledChart = styled('iframe')`
+    ${({ chartWidth }) => `width: min(100%, ${chartWidth}px)`};
+    ${({ customAlign }) => !customAlign && `float: ${customAlign};`};
+`;
+
+const Chart = ({ nodeData: { options } }) => {
+    const chartHostUrl = options.url;
```

Comment:
nit it seems a little redundant to create `const`s for some of these properties that we just pass to a function or as a prop.